samples/subsys/logging/syst/sample.yaml

```diff
@@ -1,18 +1,36 @@
 sample:
   name: syst sample
 tests:
-  sample.logger.syst.v1:
-    platform_allow: mps2_an385 sam_e70_xplained
+  sample.logger.syst.v1.deferred:
+    extra_args: CONF_FILE=prj_v1_deferred.conf
```

Comment:
why not use `OVERLAY_CONFIG` here and keep all common stuff in `prj.conf`. Now there is a lot of redundancy in those config files.

Reply:
OVERLAY! That's the keyword I was looking for. Thanks. Updated.